spec/features/admin_can_edit_an_item_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.